multinet/api/views/workspace.py

```diff
@@ -49,6 +49,21 @@ def create(self, request):
         assign_perm('owner', request.user, workspace)
         return Response(WorkspaceSerializer(workspace).data, status=status.HTTP_200_OK)
 
+    @swagger_auto_schema(
+        request_body=WorkspaceRenameSerializer(),
+        responses={200: WorkspaceSerializer()},
+    )
+    @method_decorator(permission_required_or_403('owner', (Workspace, 'name', 'name')))
```

Comment:
According to the [permissions RFC](https://github.com/multinet-app/multinet-rfcs/tree/master/0005-permissions_model#roles), this should be a `maintainer` level action. @AlmightyYakob do I have that right?